stuff_to_do_data.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+
+STUFF_TO_DO = [
```

Comment:
I'm a bit torn about whether this deserves to be in this "database" of sorts, or whether it should be treated more like dumb text in the template. Like, will the hotels be a database too?

If we wanted to move it to the template, there might be a way to use write a function that abstracts the common html wrappers.